### PR TITLE
Ana/5119 dont encourage proposal creation polkadot

### LINKED ISSUE #5119: the message when polkadot / kusama have no proposals is incorrect
**Describe the bug**

if there are no proposals for these networks we display a message that encourages the user to create a proposal - but this is not possible on these networks yet.

**Expected behavior**

let's change the message to exclude the part about creating a new proposal in lunie.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #5119

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f89d629590e58009dd5c185--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
E2E currently fail all of them because of poor production API performance

**Comment by Bitcoinera** [comment]
Ready to be merged

File: app/src/components/governance/PageProposals.vue [hunk 1]
```diff
@@ -64,8 +64,13 @@
         <TmDataMsg icon="gavel">
           <div slot="title">No Governance Proposals</div>
           <div slot="subtitle">
-            There are currently no governance proposals to display. Click the
-            'Create Proposal' button to submit the first network proposal!
+            There are currently no governance proposals to display.
+            {{
+              currentNetwork.network_type === `polkadot`
```
Comment (jbibla) [inline app/src/components/governance/PageProposals.vue:69]: i probably would have `v-if`'d a template or p tag instead - but not a big deal

**Review by jbibla — APPROVED** [review]

**Review by faboweb — APPROVED** [review]

**Pull request merged**